.rspec

```diff
@@ -1,2 +1,3 @@
 --color
 --require spec_helper
+--format documentation
```

Comment:
This allows more input in the terminal when running our test suite: `Use the documentation formatter to see the documentation strings passed to describe it`